docs/components/Icon/Icon.stories.mdx

```diff
@@ -326,6 +326,7 @@ use the default icon color.
 | Icon                   | `Name`   |
 | :--------------------- | :------- |
 | <Icon name="apple" />  | `apple`  |
+| <Icon name="bank" />   | `bank`   |
```

Comment:
Let me know if you prefer this icon on the `Payment` section

Reply:
That would be preferable!